api/delete-text.md

```diff
@@ -1,15 +1,29 @@
-# Deleting Text Data for a Metric
+# Deleting Text Data for a Metric or a Set of Metrics
 
-This API call is for deleting text data from the IRONdb cluster for a specific metric. It will remove data from the beginning of time up until the time provided by the user for that metric.
+This API call is for deleting text data from the IRONdb cluster for a specific metric or for a set of metrics (when wildcards or a tag query are specified). It will remove text data from the beginning of time up until the end time provided by the user for that metric(s).
 
-This call will return an empty array on success. If there is an error, it will return a JSON object with the error.
+The default behavior is deletion of data for all rollups, but it is possible also to specify particular rollups in the header given below, which may be used to remove data for specific rollup(s) which are not needed.
```

Comment:
No such thing as rollups for text.

Reply:
argh, copy and paste bug
rechecked everything for any more of this, hopefully it is all fixed now